Sources/SwiftlyCore/HTTPClient.swift

```diff
@@ -22,6 +22,100 @@ private func makeRequest(url: String) -> HTTPClientRequest {
     return request
 }
 
+struct SwiftOrgSwiftlyRelease: Codable {
+    var name: String
+}
+
+struct SwiftOrgPlatform: Codable {
+    var name: String
+    var platform: String
+    var archs: [String]
+
+    var platformName: String {
```

Comment:
Can we make it clear the difference between `platform` and `platformName`. In actual fact if we aren't using the `platform` variable lets not parse it.

Reply:
The platform is not necessary here. It'll be removed in the latest revision. Also, the platformName gets a comment indicating it's purpose, which is to match swiftly's PlatformDefinition.name field.